live/root/etc/systemd/system/agama-dud.service

```diff
@@ -0,0 +1,31 @@
+[Unit]
+Description=Agama DUD
+
+After=network-online.target
+
+# and after we process agama params like info which can contain password
+After=agama-cmdline-process.service
+# the DUD should replace wrong self_update
+After=agama-self-update.service
+
+# before starting the Agama servers so they use the new packages
+Before=agama-web-server.service
+Before=agama.service
+Before=x11-autologin.service
```

Comment:
this dependency will have to be removed once https://github.com/agama-project/agama/pull/1868 get merged

Reply:
Addressed in the last commit